scene/resources/material.cpp

```diff
@@ -365,6 +365,7 @@ Ref<Shader> ShaderMaterial::get_shader() const {
 }
 
 void ShaderMaterial::set_shader_parameter(const StringName &p_param, const Variant &p_value) {
+	WARN_PRINT_ED("Parameter \'" + p_param + "\' does not exist in base " + get_class_name());
```

Comment:
Make the message more explicit:

```suggestion
	WARN_PRINT_ED("Shader parameter \'" + p_param + "\' does not exist in base " + get_class_name());
```